src/current/cockroachcloud/folders.md

```diff
@@ -17,11 +13,11 @@ Folders allow you to organize and manage access to your clusters according to yo
 
 Each folder can contain a mix of folders and clusters. Roles assigned on a folder are inherited by its descendants.
 
-Each folder that you create has an ID. To create a cluster in a folder or to move a cluster into a folder, you set its `parent_id` field to the folder's ID.
+Each folder that you create has an ID. Clusters or folders within a folder have their parent ID (`parent_id` if using the CockroachDB {{ site.data.products.cloud }} API) set to that folder's ID.
 
-If no parent ID is specified for a cluster, it is created at the root level of the organization. Clusters and folders at the organization level have their `parent_id` set to `root`.
+If a cluster or folder has no parent ID, or if its parent ID is set to `root`, its location is the root level of the organization.
 
-To move a folder or a cluster, you update its `parent_id`. If you move a folder that contains descendant resources, the descendant resources are not directly modified. A folder operation may fail in the following circumstances:
+To move a folder or a cluster, you update its parent ID. If you move a folder that contains descendant resources, the descendant resources are not directly modified. A folder operation may fail in the following circumstances:
```

Comment:
Not for this PR but "the descendant resources are not directly modified" is ... very computer speak 

like are they indirectly modified? what does this mean in practice?  confusing

my mind translates this as "the root of the subtree has a new parent ID and nothing else is changed" - basically a pointer from root of subtree points at new parent ID - is that what this means? 

extremely computer-ish 

PS anyway feel free to ignore this comment for now if you just wanna merge this

Reply:
Thanks for the prompt to rewrite this.